bucket/vscode.json

```diff
@@ -52,6 +52,11 @@
         "if (!(Test-Path \"$dir\\data\\user-data\") -and (Test-Path \"$env:AppData\\Code\")) {",
         "    info '[Portable Mode] Copying user data...'",
         "    Copy-Item \"$env:AppData\\Code\" \"$dir\\data\\user-data\" -Recurse",
+        "}",
+        "$extensions_file = \"$dir\\data\\extensions\\extensions.json\"",
```

Comment:
I would suggest changing `\"$dir\"` to `\"$persist_dir\"`, since the **data** folder will always be in the persist directory, not the `app\current` directory